src/encoded/static/components/datacolors.js

```diff
@@ -221,7 +221,14 @@ class DataColors {
     //         integer -- do not pass a string with a "%" sign at the end.
     colorList(keys, options) {
         let colors = [];
-        if (keys && keys.length > 0) {
+        if (options && keys && options.merryGoRoundColors && keys.length > 0) {
+            // Map the given keys to colors consistently
+            colors = keys.map((key, index) => {
+                const i = index % rootColorList.length;
+                const outColor = rootColorList[i % rootColorList.length];
```

Comment:
Why are there two chained mod operations here? I think the second mod will always produce the same value as `i` has itself — `i` always equals the result of `i % rootColorList.length` if `i` is less than rootColorList.length, and `i` is always less than rootColorList.length because of the first mod operation.